optaplanner-core/src/test/java/org/optaplanner/core/impl/testdata/domain/clone/deepcloning/TestdataDeepCloningEntity.java

```diff
@@ -68,6 +72,8 @@ public void setValue(TestdataValue value) {
     }
 
     @DeepPlanningClone
+    @CustomShadowVariable(sources = {@PlanningVariableReference(variableName = "value")},
```

Comment:
Why this update to `TestData`, when there is no update to any of the actual tests?

Reply:
the field is called shadowVariable - but it wasn't one already. Just makes more sense like this.